README.md

```diff
@@ -29,18 +29,21 @@ For a more comprehensive overview of the resources that come together to build,
 In addition to the apps above, IPFS GUIs can be considered to include aspects of all of the following repos:
 
 - **[ipfs/ipfs-gui](https://www.github.com/ipfs/ipfs-gui)**: This repo, used for overall planning and cross-repo work
-- **[ipfs/dir-index-html](https://www.github.com/ipfs/dir-index-html)**: Directory listing HTML for go-ipfs gateways
 - **[ipfs/distributions](https://www.github.com/ipfs/distributions)**: Source code for https://dist.ipfs.io
-- **[ipfs/i18n](https://www.github.com/ipfs/i18n)**: Internationalization work across all IPFS projects
 - **[ipfs/public-gateway-checker](https://www.github.com/ipfs/public-gateway-checker)**: Source code for https://ipfs.github.io/public-gateway-checker/
-- **[ipfs-shipyard/ipfs-companion](https://www.github.com/ipfs-shipyard/ipfs-companion)**: Browser extension that simplifies access to IPFS resources on the Web
+- **[ipfs/ipfs-companion](https://github.com/ipfs/ipfs-companion)**: Browser extension that simplifies access to IPFS resources on the Web
+- **[ipfs/ipld-explorer-components](https://github.com/ipfs/ipld-explorer-components)**: React components for https://explore.ipld.io
+- **[ipfs/ipfs-webui](https://github.com/ipfs/ipfs-webui)**: Browser front-end for IPFS nodes
+- **[ipfs/ipfs-desktop](https://github.com/ipfs/ipfs-desktop)**: An unobtrusive and user-friendly desktop application for IPFS on Windows, Mac and Linux
+- **[ipld/explore.ipld.io](https://github.com/ipld/explore.ipld.io)**: Source code for https://explore.ipld.io
+- **[ipfs-shipyard/i18n](https://github.com/ipfs-shipyard/i18n)**: Internationalization work across all IPFS projects
```

Comment:
do you know why this moved from ipfs to ipfs-shipyard?

Reply:
I believe this was a security decision (signing/publishing secrets on CI, we audit `ipfs/` org more closely.